src/render_dim.rs

```diff
@@ -47,17 +47,18 @@ pub mod r3d {
             layout: &MeshVertexBufferLayout,
         ) -> Result<RenderPipelineDescriptor, SpecializedMeshPipelineError> {
             let mut shader_defs = Vec::new();
-            shader_defs.push("LINES_3D".to_string());
+            shader_defs.push("LINES_3D".into());
             if depth_test {
-                shader_defs.push("DEPTH_TEST_ENABLED".to_string());
+                shader_defs.push("DEPTH_TEST_ENABLED".into());
             }
 
             let vertex_buffer_layout = layout.get_layout(&[
                 Mesh::ATTRIBUTE_POSITION.at_shader_location(0),
                 Mesh::ATTRIBUTE_COLOR.at_shader_location(1),
             ])?;
             let (label, blend, depth_write_enabled);
-            if key.contains(MeshPipelineKey::TRANSPARENT_MAIN_PASS) {
```

Comment:
I'm not sure if I have to specialize this for each `BLEND_*` mode now. (ALPHA, MULTIPLY, OPAQUE, and PREMULTIPLIED_ALPHA)

Reply:
I'm not entirely sure.  It should just be for the pass that the lines shader uses right, which should be just `BLEND_ALPHA`.

The bevy pull request looks like they just do `BLEND_ALPHA`, so should be good.  Not sure if they have toggle-able depth testing.
```rust
            fragment: Some(FragmentState {
                shader: self.shader.clone_weak(),
                shader_defs: vec![],
                entry_point: "fragment".into(),
                targets: vec![Some(ColorTargetState {
                    format: TextureFormat::bevy_default(),
                    blend: Some(BlendState::ALPHA_BLENDING),
                    write_mask: ColorWrites::ALL,
                })],
            }),
```